components/tyk-gateway/templates/_helpers.tpl

```diff
@@ -23,6 +23,23 @@ If release name contains chart name it will be used as a full name.
 {{- end }}
 {{- end }}
 
+{{- define "tyk-gateway.gwproto" -}}
+{{- if .Values.gateway.gwTls -}}
+https
+{{- else -}}
+http
+{{- end -}}
+{{- end -}}
+
+{{- define "tyk-gateway.dash_proto" -}}
+{{- if .Values.gateway.tls.enabled -}}
```

Comment:
I think this value controls tls for gateway instead of Dashboard.

Reply:
I deleted this alltogether